lms/static/js/verify_student/views/step_view.js

```diff
@@ -20,9 +22,9 @@
 
             /* Mix non-conflicting functions from underscore.string
              * (all but include, contains, and reverse) into the
-             * Underscore namespace
+             * Underscore namespace.
              */
-            _.mixin( _s.exports() );
+             _.mixin(_.str.exports());
```

Comment:
I need to fix the indenting here (will address with code review feedback).

Note that this mixin approach is something we should move away from in the future.